src/sentry/api/urls.py

```diff
@@ -2364,6 +2365,11 @@ def create_group_urls(name_prefix: str) -> list[URLPattern | URLResolver]:
         r"^(?P<organization_id_or_slug>[^\/]+)/(?P<project_id_or_slug>[^\/]+)/keys/(?P<key_id>[^\/]+)/stats/$",
         ProjectKeyStatsEndpoint.as_view(),
     ),
+    re_path(
+        r"^(?P<organization_id_or_slug>[^\/]+)/(?P<project_id_or_slug>[^\/]+)/keys/(?P<key_id>[^\/]+)/configuration/$",
```

Comment:
Is the hierarchy here correct? Does configuration belong to a key? If not, key should probably be a query param and url be like ` r"^(?P<organization_id_or_slug>[^\/]+)/(?P<project_id_or_slug>[^\/]+)/configurations/$"`  which would list all configurations

Reply:
Hierarchy is correct.  We've not spec'd a list endpoint for configurations.  That will come in some later unit of work.

Reply:
Is there a one to one relation between key and configuration?